### ISSUE TITLE
Problem with a particular scientific name

### ISSUE THREAD

**Issue by albenson-usgs** [usgs-bcb/tir#15]
I discovered today that there is an issue with how the ITIS processor is matching the names:
_Vermivora pinus
Vermiforma pinus_

These should both be matched to _Vermivora cyanoptera_. They are synonyms for that valid name for the blue-winged warbler. What this means in the SGCN data is that it looks like the [blue winged warbler](https://www1.usgs.gov/csas/swap/species_view.html?sciname=Vermivora%20cyanoptera) is a completely new species in the 2015 lists and that "[blue winged warbler](https://www1.usgs.gov/csas/swap/species_view.html?sciname=Setophaga%20pinus)" was dropped from many lists. Somehow we need the process to only match Dendroica pinus with Setophaga pinus and NOT match Vermiforma pinus or Vermivora pinus with Setophaga pinus.

**Comment by albenson-usgs** [comment]
Note that ITIS does not have _Vermivora pinus_ or _Vermiforma pinus_ as synonyms in their ITIS report which would make this matching more seamless https://www.itis.gov/servlet/SingleRpt/SingleRpt?search_topic=TSN&search_value=950011#null.

**Comment by skybristol** [comment]
I'm not sure how to get to the result you want here. The initial queries for these names that the ITIS processor runs are the following:

http://services.itis.gov/?wt=json&q=nameWOInd:Vermivora\%20pinus
http://services.itis.gov/?wt=json&q=nameWOInd:Vermiforma\%20pinus~0.5

"Vermiforma pinus" finds "Vermivora pinus" with a fuzzy match.

That results in a single record flagged as invalid for usage. The code then follows the acceptedTSN.

http://services.itis.gov/?wt=json&q=tsn:950045

That gives us Setophaga pinus.

I don't see any way in the ITIS data to get from Vermivora pinus to Vermivora cyanoptera. As you noted, they don't show up here at all:

http://services.itis.gov/?q=tsn:950011&wt=json

I can't even seem to get there with fuzzy matching common name:

http://services.itis.gov/?q=vernacular:Blue-winged\%20Warbler~0.5&rows=10&wt=json

Running a common name search via the ITIS web app on "Blue-winged Warbler" does return Vermivora cyanoptera as a valid species, so I don't know why the configuration on the API search is not working. Could be an issue in the index somewhere.

It seems like what you are suggesting is that ITIS is wrong or incomplete for this species. I'm not seeing an elegant fix. Essentially, you (or some data manager) would have to annotate a match that would in other cases be valid (following ITIS accepted TSN), indicating that it should be reexamined. We would then do something like try to find a match with common name, assuming we could get that search to work.

Am I missing something? Where in the ITIS data returned from the API do you see a way to get from Vermivora pinus to Vermivora cyanoptera?

**Comment by albenson-usgs** [comment]
Yes, this seems like a limitation of the ITIS database that we need to contact ITIS about. I'll work on that.
